### PR TITLE
KO: translation for ko/beta/guide/keras/index.md

### PR DESCRIPTION
**Pull request by JKIsaacLee**
KO: translation for ko/beta/guide/keras/index.md

### CODE REVIEW

File: site/ko/beta/guide/keras/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# 케라스(Keras)
+
+`tf.keras`는 TensorFlow의 딥 러닝 모델 설계와 훈련을 위한 고수준(high-level) API입니다. 이는 빠른 프로토타이핑, 최첨단 기술의 연구 및 생산에 사용되며, 세 가지 주요 이점이 있습니다:
```
Comment (cre8tor) [inline site/ko/beta/guide/keras/index.md]: 딥 러닝 -> 딥러닝
remove white space for consistency :)

File: site/ko/beta/guide/keras/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# 케라스(Keras)
+
```
Comment (cre8tor) [inline site/ko/beta/guide/keras/index.md]: Please add a disclaimer :)

File: site/ko/beta/guide/keras/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+[공식 영문 문서](https://www.tensorflow.org/?hl=en)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이
+있다면 [tensorflow/docs](https://github.com/tensorflow/docs) 깃헙 저장소로 풀 리퀘스트를 보내주시기
+바랍니다. 문서 번역이나 리뷰에 참여하려면
+[docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ko)로
+메일을 보내주시기 바랍니다.
+
+`tf.keras`는 TensorFlow의 딥러닝 모델 설계와 훈련을 위한 고수준(high-level) API입니다. 이는 빠른 프로토타이핑, 최첨단 기술의 연구 및 생산에 사용되며, 세 가지 주요 이점이 있습니다:
+
+-   *사용자 친화적*<br> Keras는 일반 사용 사례에 최적화된 간단하고 일관적인 인터페이스를 제공합니다. 이는 사용자 오류에 대해 명확하고 실용적인 피드백을 제공합니다.
+-   *모듈식 및 구성가능*<br> Keras 모델은 거의 무제한으로 구성 가능한 설계 블록과 함께 연결함으로써 생성됩니다.
```
Comment (Choiuijin1125) [inline site/ko/beta/guide/keras/index.md]: I think the author wants to tell that Keras models can be modularization. and for easy reading, I suggest '모듈식 및 구성가능' → '모듈화 및 구성가능성'

And 'building block' is often translated '구성 요소'.
So I suggest, 
'Keras 모델은 거의 무제한으로 구성 가능한 설계 블록과 함께 연결함으로써 생성됩니다.' →
'Keras 모델은 약간의 제한사항과 함께, 설정 가능한 구성 요소를 연결하여 생성됩니다.'

File: site/ko/beta/guide/keras/index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+-   *쉬운 확장*<br> 연구를 위한 새로운 아이디어를 표현하기 위해 사용자 정의 설계 블록을 작성하세요. 새로운 층(layers), 메트릭(metrics), 손실 함수를 생성하고 최첨단 모델을 개발하세요.
```
Comment (Choiuijin1125) [inline site/ko/beta/guide/keras/index.md]: For the same reason, I suggest 
'사용자 정의 설계 블록을' → '사용자 정의 구성 요소를' 

And you can translate 'metric' as '지표' or '평가지표' :)

File: site/ko/beta/guide/keras/index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+[케라스(Keras): 빠른 개요](./overview.ipynb) 가이드는 시작하는 데 도움이 될 것입니다.
+
+`tf.keras'를 통한 머신 러닝에 입문하는 분들을 위한 안내는 [입문 튜토리얼 세트](https://www.tensorflow.org/tutorials/keras)를 보세요.
+
+API에 대해 더 자세히 알아보고 싶다면, TensorFlow Keras 파워 유저의 숙지 사항을 다루는 다음 가이드 세트를 참조하세요:
```
Comment (Choiuijin1125) [inline site/ko/beta/guide/keras/index.md]: I think it's better don't use Chinese letter, If we can. 
So I just suggest  '파워 유저의 숙지 사항을 다루는' → '고급 사용자가 알아야 할 사항을 다루는' 
But '고급' is a Chinese letter lol.

File: site/ko/beta/guide/keras/index.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+-   [Keras의 기능적 API 가이드](./functional.ipynb)
```
Comment (Choiuijin1125) [inline site/ko/beta/guide/keras/index.md]: Functional API is often translated as '함수형 API'('케라스 창시자에게 배우는 딥러닝', '함수형 프로그래밍' or something like this. So I suggest '기능석 API' → '함수형 API' :)

File: site/ko/beta/guide/keras/index.md (hunk 3 above)
Comment (JKIsaacLee) [reply site/ko/beta/guide/keras/index.md]: I think 거의 무제한으로 is more natural than 약간의 제한사항과 함께. So I unchanged it.
If someone thinks it's not natural, comment plz. I'll change it.

Comment (rickiepark) [reply site/ko/beta/guide/keras/index.md]: I guess that model is 'with few restrictions', not configuration.
I use '제한없이' more than '무제한'. How about you? :)
So I suggest another one. :)
'Keras 모델은 구성 요소의 설정을 바꾸어 연결하는 식으로 거의 제한없이 만들 수 있습니다.'

Comment (JKIsaacLee) [reply site/ko/beta/guide/keras/index.md]: I modified this sentence to "케라스 모델은 구성 요소의 설정에 의해 연결되는 식으로 거의 제한없이 만들 수 있습니다.".
How about this?

Comment (rickiepark) [reply site/ko/beta/guide/keras/index.md]: Pretty Good.

I just tried to add a clue for '제한없이 만들 수 있다'. :)
When I read it repeatedly, '~에 의해 ~으로' is a little hard.
"케라스 모델은 구성 요소의 설정에 의해 연결되기 때문에 거의 제한없이 만들 수 있습니다."
or
"케라스 모델은 구성 요소의 설정에 맞춰 연결하는 식으로 거의 제한없이 만들 수 있습니다."

But your sentence is enough to approve. Thank you! :)
